rpi-eeprom-update-default

```diff
@@ -3,6 +3,11 @@ FIRMWARE_ROOT=/lib/firmware/raspberrypi/bootloader
 FIRMWARE_RELEASE_STATUS="critical"
 FIRMWARE_IMAGE_DIR="${FIRMWARE_ROOT}/${FIRMWARE_RELEASE_STATUS}"
 FIRMWARE_BACKUP_DIR="/var/lib/raspberrypi/bootloader/backup"
-BOOTFS=/boot
 USE_FLASHROM=0
 EEPROM_CONFIG_HOOK=
+
```

Comment:
Would it be better to just remove BOOTFS from rpi-eeprom-update-default then do then for loop in the script if BOOTFS is not defined?
That would still allow people to supply a default value.